cluster/rpc.go

```diff
@@ -16,12 +16,22 @@ type MapShardRequest struct {
 	pb internal.MapShardRequest
 }
 
-func (m *MapShardRequest) ShardID() uint64  { return m.pb.GetShardID() }
-func (m *MapShardRequest) Query() string    { return m.pb.GetQuery() }
+// ShardID of the storage
+func (m *MapShardRequest) ShardID() uint64 { return m.pb.GetShardID() }
+
+// Query returns the protocol buffers' query string
+func (m *MapShardRequest) Query() string { return m.pb.GetQuery() }
+
+// ChunkSize - proto buff's chunk size
 func (m *MapShardRequest) ChunkSize() int32 { return m.pb.GetChunkSize() }
 
-func (m *MapShardRequest) SetShardID(id uint64)         { m.pb.ShardID = &id }
-func (m *MapShardRequest) SetQuery(query string)        { m.pb.Query = &query }
+// SetShardID sets the shard id
+func (m *MapShardRequest) SetShardID(id uint64) { m.pb.ShardID = &id }
```

Comment:
Ditto. I think the comments should reference the "Shard map request". A "shard map request" in our system is a request to access a shard on another node.